core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -765,7 +766,7 @@ public void addProperty(JobProperty<? super P> jobProp) throws IOException {
     }
 
     @Override
-    @RequirePOST
+    @POST
```

Comment:
FTR this one is good because the `super.doConfigSubmit` uses the `req.getSubmittedForm` in [Job](https://github.com/jenkinsci/jenkins/blob/0e62b68012282648377cfac4fe6d01956d7c1f11/core/src/main/java/hudson/model/Job.java#L1328).